### ISSUE TITLE
Copy List of Muted Sounds?

### ISSUE THREAD

**Issue by Ampersammich** [funkydude/MuteSoundFile#22]
Not really an [issue], but I'm setting up a new computer, and want to copy my muted sounds from one to the other. Is there a way to access the list via notepad or something, rather than going in-game and writing down all my muted sound IDs?

**Comment by AlexFolland** [comment]
You can copy your entire WTF folder if you just want the same configuration for everything on the new machine.

**Comment by funkydude** [comment]
Indeed, copying the WTF folder is the only option. Going to leave this open as a suggestion for in-game copy functionality.
